uefi-test-runner/src/proto/console/stdout.rs

```diff
@@ -33,9 +34,14 @@ fn change_color(stdout: &mut Output) {
 fn center_text(stdout: &mut Output) {
     // Move the cursor.
     // This will make this `info!` line below be (somewhat) centered.
-    stdout.enable_cursor(true).expect("Failed to enable cursor");
+    stdout.enable_cursor(true).unwrap_or_else(|s| match s {
+        Status::Unsupported => info!("Cursor visibility control unavailable"),
+        _ => panic!("Failed to show cursor")
+    });
     stdout.set_cursor_position(24, 0).expect("Failed to move cursor");
-    stdout.enable_cursor(false).expect("Failed to disable cursor");
-
     info!("# uefi-rs test runner");
+    stdout.enable_cursor(false).unwrap_or_else(|s| match s {
+        Status::Unsupported => info!("Cursor visibility control unavailable"),
+        _ => panic!("Failed to hide cursor")
+    });
```

Comment:
In addition to fixing the bug in #1, this also ensures that the new debug messages pointing out that showing and hiding the cursor is not supported will not corrupt the display.